client/src/app/components/main-nav/main-nav.component.css

```diff
@@ -11,7 +11,7 @@
 }
 
 .mat-toolbar.mat-primary {
-  position: sticky;
+  position: fixed;
```

Comment:
The div surrounding `router-outlet` caused the nav bar not to stick to the top of the page. This fixed that